org.cataclysmdda.CataclysmDDA.yml

```diff
@@ -0,0 +1,41 @@
+id: "org.cataclysmdda.CataclysmDDA"
+runtime: "org.freedesktop.Platform"
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+command: launch-cataclysm-dda
+finish-args: 
+  - --persist=cataclysm-dda
```

Comment:
again, why do you need the persist given all the back and forth in the review you said it writes in the correct XDG location.

Reply:
Without this argument, it is impossible to create the necessary configuration directories for the game. In the [launch-cataclysm-dda](https://github.com/flathub/flathub/pull/5488/files/5070099a24ff0d51efcd2736fc6c1650e04072e8#diff-1ca570f7b28f6aa19fde4885f5c02890a6e2c7e7d4e95df5a9654ad6ad5e3627) script, I explicitly set the game to read and save its data in the `$HOME/cataclysm-dda` directory. Without the `--persist=cataclysm-dda` flag, this directory was not being created, and the game couldn't save its data there. By default, if the save directory isn't explicitly set, the game tries to save in its own directory where the binary is located, which is `/app/cataclysm-dda`. However, this directory is read-only and cannot be used for saving data.

Reply:
Maybe you fix  the launch script. https://github.com/flathub/flathub/pull/5488/files#diff-1ca570f7b28f6aa19fde4885f5c02890a6e2c7e7d4e95df5a9654ad6ad5e3627

It previously used XDG_HOME but you changed it in https://github.com/flathub/flathub/pull/5488/commits/ddfbbe2ca2383237d6ce841aafc6881a269adba0